### PR TITLE
Created UPASS card and added upass api call

### PR DESCRIPTION
**Pull request by j0pp**

### CODE REVIEW

**Comment by fanglinfang** [comment]
It'd be great if you could add a test formapState with the API data:  {"status_message": "b\"<p><span class='highlight'>Your U-PASS is not current.</span></p><p><a href='http://www.washington.edu/u-pass'>Learn more</a> about U-PASS program member benefits.</p>\\n\"", "is_current": false, "is_employee": false, "is_student": true, "in_summer": true}

**Review by fanglinfang — COMMENTED** [review]
only show the card to employee or students

**Review by fanglinfang — APPROVED** [review]

File: myuw/static/vue/components/accounts/upass.vue [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    }),
+    ...mapState({
+      employee: (state) => state.user.affiliations.employee,
+      student: (state) => state.user.affiliations.student,
+      tacoma: (state) => state.user.affiliations.tacoma,
+      bothell: (state) => state.user.affiliations.bothell,
+      seattle: (state) => state.user.affiliations.seattle,
+      pce: (state) => state.user.affiliations.pce,
+      display_activation: (state) => state.upass.value.display_activation,
+      in_summer: (state) => state.upass.value.in_summer,
```
Comment (SarthakSingh31) [inline myuw/static/vue/components/accounts/upass.vue:112]: eslint should have complained about this, we are doing camel case for our variable names

Comment (fanglinfang) [reply myuw/static/vue/components/accounts/upass.vue:112]: Nope, we may want to let eslint check the camel  case.

**Pull request merged**